test/TestInvestorLedger.sol

```diff
@@ -0,0 +1,73 @@
+pragma solidity ^0.4.11;
+
+import "truffle/Assert.sol";
+import "truffle/DeployedAddresses.sol";
+import "../contracts/loans/InvestorLedger.sol";
+import "../contracts/tokens/PrintableToken.sol";
+
+
+contract TestInvestorLedger {
+    uint256 totalLoanNeeded = 100;
+    uint16 interestPermil = 200;
+    uint256 empty = 0;
```

Comment:
I don't think this variable is necessary.

Reply:
deleted